source/common/tracing/http_tracer_impl.h

```diff
@@ -38,6 +38,11 @@ struct Decision {
 class HttpTracerUtility {
 public:
   /**
+   * Get string representation of operation name.
```

Comment:
We are going to get more strict about @return and @param, so please fill them in.